go/test/endtoend/multipooler_manager_test.go

```diff
@@ -476,7 +479,7 @@ func initializeStandby(t *testing.T, primaryPgctld *ProcessInstance, standbyPgct
 	if err != nil {
 		return fmt.Errorf("failed to create standby pooler client: %w", err)
 	}
-	queryResp, err := standbyPoolerClient.ExecuteQuery("SELECT pg_is_in_recovery()", 1)
+	queryResp, err := standbyPoolerClient.ExecuteQuery(context.Background(), "SELECT pg_is_in_recovery()", 1)
```

Comment:
Worth having/reusing a context with a timeout? To keep these one-liners, could add a helper that registers the `cancel` with `t.Cleanup()` so setting timeouts could be one-liners like `ExecuteQuery(withDeadline(t, 2*time.Second), ...)`